### PR TITLE
Add support for the /prefixes/{id}/count

### PR DESCRIPTION
**Pull request by ctriant**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -113,16 +103,25 @@
             <artifactId>commons-lang3</artifactId>
             <version>${apache.common3.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>1.4.200</version>
+        </dependency>
```
Comment (fbasios) [inline api/pom.xml:110]: Do we need the h2 dependency?

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <artifactId>pcc</artifactId>
     <packaging>pom</packaging>
     <name>pcc</name>
-    <version>${revision}</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (fbasios) [inline pom.xml:9]: In my view, we should use the `revision` as placeholder for the version in our pom files. Please check [this](https://maven.apache.org/maven-ci-friendly.html).

File: api/pom.xml (hunk 1 above)
Comment (ctriant) [reply api/pom.xml:110]: nope, it's a leftover

File: pom.xml (hunk 2 above)
Comment (ctriant) [reply pom.xml]: Updated. Is this ok?

**Review by fbasios — APPROVED** [review]

**Pull request merged**